docs/website/blog/2023-02-16-dlthub-mission.md

```diff
@@ -9,4 +9,8 @@ authors:
 tags: [dlthub, mission, dlt]
 ---
 
-The mission of dltHub is to make these folks autonomous when creating and using data in their organizations.
\ No newline at end of file
+*Since 2017, the number of Python users has been increasing by millions annually. The vast majority of these people leverage Python as a tool to solve problems at work. **Our mission is to make a next generation of Python users autonomous when they create and use data in their organizations.** For this end, we are building an open source Python library called data load tool (dlt).*
```

Comment:
```suggestion
*Since 2017, the number of Python users has been increasing by millions annually. The vast majority of these people leverage Python as a tool to solve problems at work. **Our mission is to make this next generation of Python users autonomous when they create and use data in their organizations.** For this end, we are building an open source Python library called data load tool (dlt).*
```